lib/countly_config.dart

```diff
@@ -138,6 +141,17 @@ class CountlyConfig {
     return this;
   }
 
+  /// Set to true if you want to disable location tracking
+  CountlyConfig setDisableLocation() {
```

Comment:
why we are using different naming for same functions "disableLocation" in countly_flutter.data and "setDisableLocation" in countly_config.dart?

Reply:
`disableLocation` is the actual function name (That is what is used on android sdk. IOS uses `disableLocationInfo).
For CountlyConfig methods, we usually add `set` in front of the name. So, I added it here for consistency. Examples of CountlyConfig methods include `setLoggingEnabled`, `setParameterTamperingProtectionSalt`,  `setCustomCrashSegment`, `setRequiresConsent`, `setConsentEnabled`, `setHttpPostForced`, `setStarRatingTextTitle`, `setStarRatingTextMessage`, `setStarRatingTextDismiss`, `setRecordAppStartTime`, `setLocation`.

But I can use disableLocation if that is preferred.
@ArtursKadikis what do you think?

Reply:
answered in a call. let us go with "disableLocation"